lib/sneaker/presentation.rb

```diff
@@ -1,47 +1,53 @@
 # frozen_string_literal: true
 
-require 'tty-table'
 require 'tty-cursor'
-require 'pastel'
+
+require 'sneaker/presentation/csv'
+require 'sneaker/presentation/table'
 
 module Sneaker
   class Presentation
+    def self.create(stages)
+      klass = case ENV.fetch('format', '').downcase
+              when 'csv'
+                CSV
+              else
+                Table
+              end
+
+      klass.new(stages)
+    end
+
     def initialize(stages)
-      @table = []
       @stages = stages.sort
       @cursor = TTY::Cursor
-      @pastel = Pastel.new
-
-      @stages.each do |stage|
-        @table << [formatted_stage(stage), @pastel.inverse('loading...')]
-      end
 
+      initialize_results
       draw
     end
 
     def set(stage, result)
-      set_in_table(stage, result)
+      @results[stage] = formatted_result(result)
 
       redraw
     end
 
     private
 
-    def formatted_stage(stage)
-      @pastel.bold(stage)
+    def initialize_results
+      @results = {}
+
+      @stages.each do |stage|
+        @results[stage] = empty_result
+      end
     end
 
-    def formatted_result(result)
-      return @pastel.on_red('!error') if result == :error
-      return @pastel.red(result)      if result == 'false'
-      return @pastel.green(result)    if result == 'true'
-      result
+    def empty_result
+      '⏳'
     end
```

Comment:
⏳ -> is a clock, is like we are waiting for something to happen. But the method name is called `empty_result`. 

Is confusing.